doc/users/dflt_style_changes.rst

```diff
@@ -1065,6 +1065,17 @@ Z-order
 
 
 
+``ScalarFormatter`` tick label formatting with offsets
+======================================================
+
+With the default of ``rcParams['axes.formatter.useoffset'] = True``,
+an offset will be used when it will save 4 or more digits.  This can
```

Comment:
unclear what you mean by save...maybe "will be used when the tick values have 4 or more digits"